src/main/java/net/mcreator/element/ModElementTypeLoader.java

```diff
@@ -74,6 +75,9 @@ public static void loadModElements() {
 		ModElementType.VILLAGERPROFESSION = register(new ModElementType<>("villagerprofession", null, VillagerProfessionGUI::new, VillagerProfession.class)).coveredOn(GeneratorFlavor.baseLanguage(JAVA));
 		ModElementType.VILLAGERTRADE = register(new ModElementType<>("villagertrade", null, VillagerTradeGUI::new, VillagerTrade.class)).coveredOn(GeneratorFlavor.baseLanguage(JAVA));
 
+		// Bedrock specific METs
+		ModElementType.BEITEM = register(new ModElementType<>("beitem", 'i', BEItemGUI::new, BEItem.class)).coveredOn(GeneratorFlavor.gamePlatform(BEDROCKEDITION));

```

Comment:
I have pushed a commit on master that makes it possible to reuse icons of other MEs